jpsreport/scripts/plot_Nt.py

```diff
@@ -0,0 +1,81 @@
+# run script without arguments for help
+import sys
+import os
+from xml.dom.minidom import parse
+import glob
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+def getParserArgs():
+        parser = argparse.ArgumentParser(description='Plot N-t graph measured by method A')
+        parser.add_argument("-f", "--inifile", help='jpsreport inifile (required)', required=True)
+        parser.add_argument("-i", "--id", help='id of the measurement area (optional)', nargs='?', default=-1, type=int)
+        args = parser.parse_args()
+        return args
+
+def plot_nt(measurement_id):
+    """
+    plot N(t) curves (Method A)
+    args:
+    measurement area id (int)
+    """
+    legend = False
+    fig = plt.figure(figsize=(16, 16), dpi=100)
+    ax1 = fig.add_subplot(111,aspect='auto')
+    plt.rc("font", size=30)
+    plt.rc('pdf',fonttype = 42)
+    for f in nt_files:
+        data = np.loadtxt(f)
+        Id = int(f.split("id_")[-1].split(".")[0])
+        if measurement_id == -1:
+            plt.plot(data[:, 0], data[:, 1], label="{}".format(Id))
+            legend = True
+            continue
+        
+        if Id == measurement_id:
+            plt.plot(data[:, 0], data[:, 1])
+
+    plt.xlabel("$t$", size=18)
+    plt.ylabel("$N$", size=18)
```

Comment:
Do we need the size definition?

Reply:
I will resolve this conservation. My explanation of this trivial question can be found in the comment to line 26.